src/main/scala/org/broadinstitute/dsde/agora/server/AgoraConfig.scala

```diff
@@ -8,6 +8,24 @@ import org.broadinstitute.dsde.agora.server.model.AgoraEntityType.EntityType
 
 object AgoraConfig {
   private val config: Config = ConfigFactory.load()
+  
+  val LocalEnvironment = "local"
+  val DevEnvironment = "dev"
+  val CiEnvironment = "ci"
+  val ProdEnvironment = "prod"
+
+  val Environments = List[String](LocalEnvironment, DevEnvironment, CiEnvironment, ProdEnvironment)
+
+  // TODO - what is the default??
+  lazy val environment = config.as[Option[String]]("environment").getOrElse(LocalEnvironment)
+  if (!Environments.contains(AgoraConfig.environment))
+    throw new IllegalArgumentException("Illegal environment '" + AgoraConfig.environment + "' specified.")
+
+  val envConfig = config.getConfig(environment)
+  val mockAuthenticatedUserEmail = envConfig.as[Option[String]]("mockAuthenticatedUserEmail").getOrElse("noone@broadinstitute.org")
+  def getUseOpenAMAuthenticationForEnvironment(environment: String) : Boolean = {
```

Comment:
Perhaps we could shorten / change to something like isUseOpenAmAuth? The "for environment" part is implied because environment is a parameter. Similarly, getUseGcsAuthorizationProviderForEnvironment could perhaps be isUseGcsAuth or something like that.